src/pending_transactions.erl

```diff
@@ -176,3 +177,11 @@ collect_transactions_from_player(TxMap, Nonce, TransactionListAcc) ->
 	    %% done
 	    TransactionListAcc
     end.
+
+get_status_info(_PendingTx = #pending_tx{player_map = PlMap}) ->
```

Comment:
maybe just rename these status functions to
get_num_pending_transactions... etc

or get_potato_status_info is also acceptable

Reply:
yeah, just wanted them to be more uniform; maybe a more targeted call is more appropriate idk